src/field_10x26.h

```diff
@@ -9,15 +9,22 @@
 
 #include <stdint.h>
 
+/** Implements arithmetic modulo FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFF FFFFFFFE FFFFFC2F,
+ *  represented as 10 uint64_t's in base 2^26, least significant first. Note that the limbs are allowed to
+ *  contain >26 bits each. */
 typedef struct {
     /* X = sum(i=0..9, n[i]*2^(i*26)) mod p
      * where p = 2^256 - 0x1000003D1
      */
     uint32_t n[10];
-#ifdef VERIFY
-    int magnitude;
-    int normalized;
-#endif
+    /*
+     * Magnitude m implies n[i] <= 2*m*0x3FFFFFF for i=0..8, and
+     * n[9] <= 2*m*0x3FFFFF (a factor 16 smaller).
```

Comment:
nit: that wording sounds to me like `0x3FFFFF*16 = 0x3FFFFFF` but that's false.

Reply:
Done something like this (and now also placed it in the right file).